Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -960,7 +1031,8 @@ public enum InGameICChatType : byte
 public enum InGameOOCChatType : byte
 {
     Looc,
-    Dead
+    Dead,
+    HiveXeno
```

Comment:
OOC Type?
I feel this should be moved and used the same way "Telepathic" does.
Or even hook it.